makecrx.sh

```diff
@@ -115,8 +115,19 @@ crmagic_hex="4372 3234" # Cr24
 version_hex="0200 0000" # 2
 pub_len_hex=$(byte_swap $(printf '%08x\n' $(ls -l "$pub" | awk '{print $5}')))
 sig_len_hex=$(byte_swap $(printf '%08x\n' $(ls -l "$sig" | awk '{print $5}')))
+
+# Case-insensitive matching is a GNU extension unavailable when using BSD sed.
+if [[ "$(sed --version 2>&1)" =~ "GNU" ]]; then
+  sed="sed"
+elif [[ "$(gsed --version 2>&1)" =~ "GNU" ]]; then
+  sed="gsed"
+else
+  echo "Cannot find a GNU sed. Please install one!"
```

Comment:
I would encourage installing GNU `sed` as a dependency in `install-dev-dependencies` rather than creating a conditional block here.  Devs really should be running that first thing, before ever building.  If we can, from the perspective of this script, make it so that `sed` is transparently the GNU version, that would be best.  If not, I suppose this block is okay.